ops/teardown_local.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo docker stop $(sudo docker ps -q)
```

Comment:
this is a pretty messy teardown.. i would be unhappy if all the running containers on my machine were brought down.. maybe in the local script, you can change the names to `near-testnet-*` and `near-studio` and then grep those here.

Reply:
Will fix it later.